gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/internal/event/EventMongoRepositoryImpl.java

```diff
@@ -64,14 +59,8 @@
         // Extract result.
         aggregationOperations.add(Aggregation.replaceRoot("doc"));
 
-        List<Criteria> criteriaList = buildDBCriteria(criteria);
-
-        // Match criteria.
-        if (!criteriaList.isEmpty()) {
-            aggregationOperations.add(Aggregation.match(new Criteria().andOperator(criteriaList.toArray(new Criteria[0]))));
-        }
-
         // Sort
+        // TODO: check if necessary
```

Comment:
TODO

Reply:
It is necessary ;) 
I removed the TODO comment